src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/URLIndexInterface.java

```diff
@@ -0,0 +1,15 @@
+package au.org.emii.geoserver.wms;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public interface URLIndexInterface {
```

Comment:
I think in most of our code we generally only uppercase the first letter of an acronym in a class name. So I think this should be UrlIndexInterface.